packages/oui/src/ScrollProgress/ScrollProgress.js

```diff
@@ -12,21 +12,53 @@ export function calculateVerticalProgress(bounds, topOffset = 0, bottomOffset =
 }
 
 const ScrollProgress = React.forwardRef(function ScrollProgress(props, ref) {
-  const { onChange, onEnter, onExit, ...other } = props
+  const { friction = 0, onChange, onEnter, onExit, precision = 0.001, ...other } = props
 
   const rootRef = React.useRef(null)
 
+  const stateRef = React.useRef({
+    currentInnerProgress: 0,
+    currentProgress: 0,
+    isEasing: false,
+    targetInnerProgress: 0,
+    targetProgress: 0,
```

Comment:
I don't really understand what `current` and `target` represents.

`target` seems to be the actual scroll progress, while `current` seems to be a modified version `target` (because of the `friction` property)..?

I wonder if we could make the distinction even clearer here.

Reply:
Yeah, as we discussed in person today; it's a bit hard to distinguish what `current` and `target` actually refers to.

Let's let it marinate a while! 😊

Reply:
My suggestion would be to go with the following:

```js
const stateRef = React.useRef({
    isEasing: false,
    rafInnerProgress: 0,
    rafProgress: 0,
    scrollInnerProgress: 0,
    scrollProgress: 0,
})

const handleRaf = () => {
  // updates raf values
}

const handleScroll = () => {
  // updates scroll values
}
```

With this change, the state names match up with the naming of the handlers, what do you think @adamsoderstrom? :)

Reply:
Ah, nice! That looks like a nice tradeoff! 🎉